src/main/scala/io/movies/api/MovieServiceHttp4s.scala

```diff
@@ -1,46 +1,44 @@
 package io.movies.api
 
 import io.movies.model._
+import io.movies.services.MovieService
 import cats.effect._
 import cats.syntax.all._
-import io.movies.services.MovieService
 import org.http4s._
 import org.http4s.dsl.Http4sDsl
 import org.http4s.server.blaze._
 import org.http4s.circe._
 import io.circe.generic.auto._
 
 class MovieServiceHttp4s[F[_]: Sync: MovieService]
-  (implicit monadThrow: MonadThrow[F])
+  //(implicit monadThrow: MonadThrow[F])
```

Comment:
**suggestion:** This line should be removed, if it's not used.

Reply:
I am missing a face palm emoji right now

Reply:
See https://github.com/gutiory/MoviesServer/pull/1/commits/bbbea420ee557f7e39602cb409c96dedc38f229e